src/test/java/com/uber/cadence/internal/testing/WorkflowTestingTest.java

```diff
@@ -775,6 +775,11 @@ public Object getCurrentContext() {
     public void setCurrentContext(Object context) {
       MDC.put("test", String.valueOf(context));
     }
+
+    @Override
+    public void unsetCurrentContext() {
+      MDC.remove("test");
```

Comment:
Nit: Maybe add an assertion that this key is removed

Reply:
sure. There is actually no easy way to assert on MDC directly, so I added a counter to ensure state is cleaned.